client/webserver/types.go

```diff
@@ -301,11 +301,17 @@ func (ord *orderReader) StatusString() string {
 		if ord.Cancelling {
 			return "cancelling"
 		}
+		if isLive {
+			return "booked/settling"
+		}
 		return "booked"
 	case order.OrderStatusExecuted:
 		if isLive {
 			return "settling"
 		}
+		if ord.Filled == 0 {
+			return "expired"
```

Comment:
Good changes in this PR, but I think expired is a bit confusing as it hints at some time limit or deadline that passed, but it was just not booked and had no matches to settle.  Some more accurate ideas: "missed", "no match", "unfilled", and combinations with a "executed / ..." prefix

Reply:
I agree that's more clear for sure. It's updated.